doc/rst/technotes/libraries.rst

```diff
@@ -403,6 +475,14 @@ program.  Each library file must include the chapel runtime and standard modules
 for its own functionality and when two or more libraries are linked to a program
 this leads to multiple definitions of these functions.
 
+LLVM
+----
+
+LLVM support with ``--library`` is currently a work-in-progress.  For the 1.19
```

Comment:
> For the 1.19 release, it is only supported for C with a subset of the flags, and does not support arrays. 

I'd suggest using:

> Currently,  it is only supported for C with a subset of the flags, and does not support arrays.

for improved maintainability, e.g. if it remains unchanged in 1.20, we don't have to modify this line, and if it gets updated soon, the master docs can reflect that before 1.20 release.

Reply:
lol, that's the opposite of what Michael wanted.  I suspect that it won't matter that much, though, since we're probably going to get LLVM in line with the C backend for the next release

Reply:
OK, I'm fine with leaving it as is then.

I think including versions where features are introduced is great in general, but generally recommend against it due to how inconsistently we apply it across the documentation.